scipy/spatial/transform/rotation.py

```diff
@@ -365,7 +365,7 @@ class Rotation(object):
     output formats supported, consult the individual method's examples.
 
     """
-    def __init__(self, quat, normalized=False, copy=True):
+    def __init__(self, quat, normalize=True, copy=True):
```

Comment:
here you're renaming a keyword, that's a backwards compat break right? or was this not in 1.3.0?

Reply:
The situation here is slightly unclear. The class was designed as such as not to use `__init__` directly and this was clearly stated in the documentation from the very beginning. 

So I suppose we can make this change without violating any "contracts". Do you agree?

Reply:
Ah, that makes sense, rings a bell. Okay, then yes I agree.